upgrade_substrate.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
It simple, but convenient. +1 for renaming it. It's silly since I don't know curl magic, if you can provide me with some info than I'm more than happy to improve the script.

Reply:
Turns out it's not even `curl` magic, just good old `git` wizardry:
```
git ls-remote https://github.com/paritytech/substrate.git | grep refs/heads/master | cut -c 1-7
```

Reply:
Changed.